src/commands/cmd_memory.c

```
@@ -11,6 +11,307 @@
+		edges_sample_size = sample_size;
+	}
+
+	// return estimated edge attribute set size
+	return edge_memory_usage;
+}
+
+static size_t _SampleVector
+(
+	const Graph *g,
+	const GrB_Vector V,
+	GxB_Iterator it,
```

Cppcheck (2.10-2)

[error] Memory leak

(memleak)